util/configure

```diff
@@ -346,6 +349,13 @@ for my $opt (@ARGV) {
         }
         $path = File::Spec->rel2abs($path);
         push @ngx_opts, "--with-$lib=$path";
+        $ngx_libs{$lib} = "y";
+
+    } elsif ($opt =~ /^--with-(openssl|pcre|zlib)-opt=(.*)/) {
+        my ($lib, $opt) = ($1, $2);
+        $opt.=" -g";
+        push @ngx_opts, "--with-$lib-opt=$opt";
+        $ngx_libs_opt{$lib."-opt"} = "y";
```

Comment:
We are not using the value `y` in any way, so a Perl boolean value `1` should be enough here?